src/realm/sync/network/websocket.cpp

```diff
@@ -1109,10 +1109,10 @@ class CloseStatusErrorCategory : public std::error_category {
     {
         // Converts an error_code to one of the pre-defined status codes in
         // https://tools.ietf.org/html/rfc6455#section-7.4.1
-        if (error_code == 1000 || error_code == 0) {
-            return ErrorCodes::error_string(ErrorCodes::OK);
+        if (error_code == 0) {
```

Comment:
I think a value of 0 is not possible anymore. This was used when ErrorCodes::OK was 0 and there was no WebSocket_OK but I see you addressed that. I think you don't need to handle this in a special way anymore.